bundler.d/development.rb

```diff
@@ -1,6 +1,8 @@
 group :development do
   gem 'maruku', '~> 0.7'
-  gem 'rubocop', '~> 0.64.0'
+  gem 'rubocop', '~> 0.71.0'
```

Comment:
BTW, maybe this should go into `bundler.d/test.rb`?

Reply:
Maybe someone had in mind that it helps with the development...
But I agree to move... isn't it here for some local requires to work? If it works after move, let's do it :)

Reply:
Then let me change this in this PR, minimizes the diff...